README.md

```diff
@@ -13,17 +13,29 @@ Your feedback, contributions, and issue reports will be integral to evolving thi
   
   1. Install general prerequisites
 
+      Debian/Ubuntu
       ```bash
       sudo apt install curl git build-essential libssl-dev pkg-config 
       ```
+      Rocky Linux/RHEL/CentOS/Fedora
+      ```bash
+      sudo dnf install curl git pkg-config zlib-devel make gcc perl openssl-devel
+      ```
 
-  2. Install Protobuf (required for gRPC)
-  
+  3. Install Protobuf (required for gRPC)
+
+      Debian/Ubuntu
       ```bash
-      sudo apt install protobuf-compiler libprotobuf-dev #Required for gRPC
+      sudo apt install protobuf-compiler libprotobuf-dev
       ```
-  3. Install the clang toolchain (required for RocksDB and WASM secp256k1 builds)
+      Rocky Linux/RHEL/CentOS
+      ```bash
+      sudo dnf install protobuf-compiler protobuf-devel

```

Comment:
`protobuf-compiler` is not available in the package registry for CentOS. Most sources online refer to building from source. Please add link to source build or revisit this part to ensure that installation instructions work across CentOS-based systems.